src/common/snippets/src/op/load.cpp

```diff
@@ -13,12 +13,15 @@ namespace snippets {
 namespace op {
 
 Load::Load(const Output<Node>& x, const size_t count, const size_t offset) : MemoryAccess({x}) {
-    constructor_validate_and_infer_types();
+    m_input_ports.resize(get_output_size());
     set_input_port_descriptor({count, offset}, 0);
```

Comment:
`set_input_port_descriptor` is a virtual member now, so we should explicitly specify the class 
`MemoryAccess::set_input_port_descriptor(...)`

Reply:
When we needed override methods, it's valid. But now as you said, we don't have to override them and make virtual them. So I removed `virtual` and left just s`et_input_port_descriptor()`. Thanks!